src/zzz_od/application/charge_plan/charge_plan_app.py

```
@@ -70,6 +73,28 @@ def goto_menu(self) -> OperationRoundResult:
+    def exchange_ether_battery(self) -> OperationRoundResult:
+        if self.ether_battery_exchange_checked:
+            return self.round_success('已尝试')
+
+        self.ether_battery_exchange_checked = True
+        if not self.config.auto_exchange_ether_battery:
+            return self.round_success(ExchangeEtherBattery.STATUS_NOT_ENABLED)
+
+        op = ExchangeEtherBattery(self.ctx)
+        op_result = op.execute()
+        status = op_result.status or '兑换以太电池结束'
+        # 兑换失败不阻断体力计划，后续统一回到菜单继续识别电量。
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)